src/Grpc.AspNetCore.Server/Internal/CallHandlers/ClientStreamingServerCallHandler.cs

```diff
@@ -31,14 +33,25 @@ internal class ClientStreamingServerCallHandler<TService, TRequest, TResponse> :
         where TService : class
     {
         private readonly ClientStreamingServerMethod<TService, TRequest, TResponse> _invoker;
+        private readonly IReadOnlyList<InterceptorRegistration> _interceptors;
+        private static readonly Func<Interceptor, ClientStreamingServerMethod<TRequest, TResponse>, ClientStreamingServerMethod<TRequest, TResponse>> BuildInvoker = (interceptor, next) =>
+        {
+            return (requestStream, context) =>
+            {
+                return interceptor.ClientStreamingServerHandler(requestStream, context, next);
+            };
+        };
 
         public ClientStreamingServerCallHandler(
             Method<TRequest, TResponse> method,
             ClientStreamingServerMethod<TService, TRequest, TResponse> invoker,
             GrpcServiceOptions serviceOptions,
-            ILoggerFactory loggerFactory) : base(method, serviceOptions, loggerFactory)
+            ILoggerFactory loggerFactory,
+            IReadOnlyList<InterceptorRegistration> interceptors) 
```

Comment:
Could the interceptors be fetched from the already passed in GrpcServiceOptions?